src/features/hotspots/setup/HotspotSetupConfirmLocationScreen.tsx

```diff
@@ -30,15 +31,22 @@ const HotspotSetupConfirmLocationScreen = () => {
   const {
     account: { account },
   } = useSelector((state: RootState) => state)
-  const { loading, result } = useAsync(loadLocationFeeData, [])
+  const { loading, result, error } = useAsync(loadLocationFeeData, [])
```

Comment:
this is where Tim was getting stuck, so I added a case to check the error, but it actually should error out sooner than this screen